assets/sass/components/global/_googlesitekit-new-metric-badge.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * NewMetricBadge styles.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-new-metric-badge {
+	background-color: $c-violet-v-500;
+	color: $c-violet-v-0;
+	margin-left: 6px;
+	text-transform: none;
+}
+
+.googlesitekit-new-metric-badge__tooltip {
+
+	&.googlesitekit-tooltip {
+
+		&.MuiTooltip-tooltip {
```

Comment:
This hierarchy doesn't work as expected because the tooltip itself is not a child of `googlesitekit-new-metric-badge__tooltip`, so none of the styles below are taking effect.

Reply:
I've reordered the classes so it makes more sense and works as expected. Basically, we're targeting `<element class="googlesitekit-tooltip MuiTooltip-tooltip googlesitekit-new-badge__tooltip">`